diff -Nur xview-3.2p1.4.orig/clients/olvwm-4.1/mem.h xview-3.2p1.4/clients/olvwm-4.1/mem.h
--- xview-3.2p1.4.orig/clients/olvwm-4.1/mem.h	Thu Jan 13 22:40:02 1994
+++ xview-3.2p1.4/clients/olvwm-4.1/mem.h	Mon Aug 19 22:48:01 2002
@@ -39,7 +39,7 @@
 #else
 #define MemNew(t) ((t *)MemAlloc((unsigned int)sizeof(t)))
 #define MemNewString(s) (strcpy((char *)MemAlloc(strlen(s)+1),s))
-#endif MEMDEBUG
+#endif /* MEMDEBUG */
 
 #ifdef OW_I18N_L4
 
@@ -49,6 +49,6 @@
 
 #define MemNewText(s)	MemNewString((char *)s)
 
-#endif OW_I18N_L4
+#endif /* OW_I18N_L4 */
 
-#endif _OLWM_MEM_H
+#endif /* _OLWM_MEM_H */
diff -Nur xview-3.2p1.4.orig/clients/olvwm-4.1/olgx_impl.h xview-3.2p1.4/clients/olvwm-4.1/olgx_impl.h
--- xview-3.2p1.4.orig/clients/olvwm-4.1/olgx_impl.h	Thu Jan 13 22:40:03 1994
+++ xview-3.2p1.4/clients/olvwm-4.1/olgx_impl.h	Mon Aug 19 22:48:01 2002
@@ -114,7 +114,7 @@
 void               xcolor_to_hsv();
 void               olgx_hsv_to_3D();
 
-#endif	!OL_PRIVATE_DEFINED
+#endif	/* !OL_PRIVATE_DEFINED */
 
 
 
diff -Nur xview-3.2p1.4.orig/clients/olvwm-4.1/st.h xview-3.2p1.4/clients/olvwm-4.1/st.h
--- xview-3.2p1.4.orig/clients/olvwm-4.1/st.h	Thu Jan 13 22:40:05 1994
+++ xview-3.2p1.4/clients/olvwm-4.1/st.h	Mon Aug 19 22:48:01 2002
@@ -64,4 +64,4 @@
 
 int st_strhash();
 
-#endif ST_INCLUDED
+#endif /* ST_INCLUDED */
diff -Nur xview-3.2p1.4.orig/clients/olwm/mem.h xview-3.2p1.4/clients/olwm/mem.h
--- xview-3.2p1.4.orig/clients/olwm/mem.h	Tue Jun 29 07:11:49 1993
+++ xview-3.2p1.4/clients/olwm/mem.h	Mon Aug 19 22:48:01 2002
@@ -36,7 +36,7 @@
 #else
 #define MemNew(t) ((t *)MemAlloc((unsigned int)sizeof(t)))
 #define MemNewString(s) (strcpy((char *)MemAlloc(strlen(s)+1),s))
-#endif MEMDEBUG
+#endif /* MEMDEBUG */
 
 #ifdef OW_I18N_L4
 
@@ -46,6 +46,6 @@
 
 #define MemNewText(s)	MemNewString((char *)s)
 
-#endif OW_I18N_L4
+#endif /* OW_I18N_L4 */
 
-#endif _OLWM_MEM_H
+#endif /* _OLWM_MEM_H */
diff -Nur xview-3.2p1.4.orig/clients/olwmslave/i18n.h xview-3.2p1.4/clients/olwmslave/i18n.h
--- xview-3.2p1.4.orig/clients/olwmslave/i18n.h	Tue Jun 29 07:12:12 1993
+++ xview-3.2p1.4/clients/olwmslave/i18n.h	Mon Aug 19 22:48:01 2002
@@ -24,8 +24,8 @@
 extern wchar_t  *ctstowcsdup();
 extern char     *ctstombsdup();
 extern char     *wcstoctsdup();
-#endif OW_I18N_L4
+#endif /* OW_I18N_L4 */
 
-#endif i18n_DEFINED
+#endif /* i18n_DEFINED */
 
 
diff -Nur xview-3.2p1.4.orig/clients/olwmslave/mem.h xview-3.2p1.4/clients/olwmslave/mem.h
--- xview-3.2p1.4.orig/clients/olwmslave/mem.h	Tue Jun 29 07:12:12 1993
+++ xview-3.2p1.4/clients/olwmslave/mem.h	Mon Aug 19 22:48:01 2002
@@ -16,4 +16,4 @@
 #define MemNewString(s) (strcpy(MemAlloc(strlen(s)+1),s))
 #ifdef OW_I18N_L4
 #define MemNewWString(s) (wscpy(MemAlloc((wslen(s)+1) * sizeof(wchar_t)),s))
-#endif OW_I18N_L4
+#endif /* OW_I18N_L4 */
diff -Nur xview-3.2p1.4.orig/clients/props/l10n_props.h xview-3.2p1.4/clients/props/l10n_props.h
--- xview-3.2p1.4.orig/clients/props/l10n_props.h	Wed Apr 19 08:53:40 1995
+++ xview-3.2p1.4/clients/props/l10n_props.h	Mon Aug 19 22:48:01 2002
@@ -70,7 +70,7 @@
 	 * format for strftime, not string itself).
 	 */
 	wchar_t			*(*convert_label)();
-#endif OW_I18N
+#endif /* OW_I18N */
 
 	int			default_value;
 
diff -Nur xview-3.2p1.4.orig/lib/libxview/attr/attr_impl.h xview-3.2p1.4/lib/libxview/attr/attr_impl.h
--- xview-3.2p1.4.orig/lib/libxview/attr/attr_impl.h	Tue Jun 29 07:14:35 1993
+++ xview-3.2p1.4/lib/libxview/attr/attr_impl.h	Mon Aug 19 22:48:01 2002
@@ -45,4 +45,4 @@
 extern int		attr_count_avlist();
 extern Attr_avlist	attr_copy_valist();
 
-#endif _attr_impl_h_already_included
+#endif /* _attr_impl_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/base/gen_impl.h xview-3.2p1.4/lib/libxview/base/gen_impl.h
--- xview-3.2p1.4.orig/lib/libxview/base/gen_impl.h	Tue Jun 29 07:14:44 1993
+++ xview-3.2p1.4/lib/libxview/base/gen_impl.h	Mon Aug 19 22:48:01 2002
@@ -39,4 +39,4 @@
     char		*instance_name;
 } Generic_info;
 
-#endif _gen_impl_h_already_included
+#endif /* _gen_impl_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/base/qualifyx.h xview-3.2p1.4/lib/libxview/base/qualifyx.h
--- xview-3.2p1.4.orig/lib/libxview/base/qualifyx.h	Tue Jun 29 07:14:40 1993
+++ xview-3.2p1.4/lib/libxview/base/qualifyx.h	Mon Aug 19 22:48:01 2002
@@ -33,4 +33,4 @@
 #define Visual		XVisual_t
 #define Window		XWindow_t
 
-#endif _view2_private_qualify_x_h_already_included
+#endif /* _view2_private_qualify_x_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/base/xv_debug.h xview-3.2p1.4/lib/libxview/base/xv_debug.h
--- xview-3.2p1.4.orig/lib/libxview/base/xv_debug.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/base/xv_debug.h	Mon Aug 19 22:48:01 2002
@@ -12,9 +12,9 @@
 #ifndef FILE
 #if !defined(SVR4) && !defined(__linux__)
 #undef NULL
-#endif SVR4
+#endif /* SVR4 */
 #include <stdio.h>
-#endif FILE
+#endif /* FILE */
 #include <xview/pkg.h>		/* needed to get definition of Xv_private */
   /* 
    * Not strictly necessary to include <stdio.h> here, but eliminates
@@ -88,7 +88,7 @@
  * xv_got_debugger to TRUE from the debugger.
  */
 
-#else _XV_DEBUG
+#else /* _XV_DEBUG */
 
 #define DEBUG_ONLY(x)
 #define AN_ERROR(expr, flag)	(expr)
@@ -97,6 +97,6 @@
 #define FATAL_ERROR(exit_code)	if (xv_abort_fatal_error) abort() \
 				else exit(exit_code)
 
-#endif _XV_DEBUG
+#endif /* _XV_DEBUG */
 
-#endif _xv_debug_h_already_included
+#endif /* _xv_debug_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/base/xv_list.h xview-3.2p1.4/lib/libxview/base/xv_list.h
--- xview-3.2p1.4.orig/lib/libxview/base/xv_list.h	Tue Jun 29 07:14:41 1993
+++ xview-3.2p1.4/lib/libxview/base/xv_list.h	Mon Aug 19 22:48:01 2002
@@ -91,4 +91,4 @@
 	for (_this = (_type)_head, _next = XV_SL_SAFE_NEXT(_this);	\
 	     _this; _this = (_type)_next, _next = XV_SL_SAFE_NEXT(_this))
 
-#endif _xv_list_h_already_included
+#endif /* _xv_list_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/base/xv_xlib.h xview-3.2p1.4/lib/libxview/base/xv_xlib.h
--- xview-3.2p1.4.orig/lib/libxview/base/xv_xlib.h	Tue Jun 29 07:14:43 1993
+++ xview-3.2p1.4/lib/libxview/base/xv_xlib.h	Mon Aug 19 22:48:01 2002
@@ -27,5 +27,5 @@
 	int             count;
 } Xv_xrectlist;
 
-#endif _view2_private_xv_xlib_h_already_included
+#endif /* _view2_private_xv_xlib_h_already_included */
 
diff -Nur xview-3.2p1.4.orig/lib/libxview/canvas/canvas.h xview-3.2p1.4/lib/libxview/canvas/canvas.h
--- xview-3.2p1.4.orig/lib/libxview/canvas/canvas.h	Tue Jun 29 07:14:46 1993
+++ xview-3.2p1.4/lib/libxview/canvas/canvas.h	Mon Aug 19 22:48:33 2002
@@ -100,9 +100,9 @@
     CANVAS_X_PAINT_WINDOW	= CANVAS_ATTR(ATTR_BOOLEAN,      65),
     CANVAS_PAINTWINDOW_ATTRS	= CANVAS_ATTR_LIST(ATTR_RECURSIVE, ATTR_AV, 70),
     CANVAS_NO_CLIPPING		= CANVAS_ATTR(ATTR_BOOLEAN,      75),
-    CANVAS_CMS_REPAINT          = CANVAS_ATTR(ATTR_BOOLEAN,      80),
+    CANVAS_CMS_REPAINT          = CANVAS_ATTR(ATTR_BOOLEAN,      80)
 #ifdef OW_I18N
-    CANVAS_IM_PREEDIT_FRAME     = CANVAS_ATTR(ATTR_OPAQUE,       85)
+  , CANVAS_IM_PREEDIT_FRAME     = CANVAS_ATTR(ATTR_OPAQUE,       85)
 #endif
 } Canvas_attribute;
 
diff -Nur xview-3.2p1.4.orig/lib/libxview/dnd/dndimpl.h xview-3.2p1.4/lib/libxview/dnd/dndimpl.h
--- xview-3.2p1.4.orig/lib/libxview/dnd/dndimpl.h	Tue Jun 29 07:15:58 1993
+++ xview-3.2p1.4/lib/libxview/dnd/dndimpl.h	Mon Aug 19 22:48:01 2002
@@ -112,4 +112,4 @@
 Pkg_private Xv_opaque	dnd_get_attr();
 Pkg_private int		dnd_destroy();
 
-#endif ~xview_dndimpl_DEFINED
+#endif /* ~xview_dndimpl_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/dnd/site_impl.h xview-3.2p1.4/lib/libxview/dnd/site_impl.h
--- xview-3.2p1.4.orig/lib/libxview/dnd/site_impl.h	Tue Jun 29 07:15:58 1993
+++ xview-3.2p1.4/lib/libxview/dnd/site_impl.h	Mon Aug 19 22:48:01 2002
@@ -81,4 +81,4 @@
 Pkg_private Xv_opaque	dnd_site_get_attr();
 Pkg_private int		dnd_site_destroy();
 
-#endif ~xview_site_impl_DEFINED
+#endif /* ~xview_site_impl_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/filter/filter.h xview-3.2p1.4/lib/libxview/filter/filter.h
--- xview-3.2p1.4.orig/lib/libxview/filter/filter.h	Tue Jun 29 07:14:57 1993
+++ xview-3.2p1.4/lib/libxview/filter/filter.h	Mon Aug 19 22:48:01 2002
@@ -38,4 +38,4 @@
 
 EXTERN_FUNCTION (void 	xv_free_filter_table, (struct filter_rec **table));
 
-#endif ~xview_filter_DEFINED
+#endif /* ~xview_filter_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/frame/fm_impl.h xview-3.2p1.4/lib/libxview/frame/fm_impl.h
--- xview-3.2p1.4.orig/lib/libxview/frame/fm_impl.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/frame/fm_impl.h	Mon Aug 19 22:48:01 2002
@@ -20,9 +20,9 @@
 #ifndef FILE
 #if !defined(SVR4) && !defined(__linux__)
 #undef NULL
-#endif SVR4
+#endif /* SVR4 */
 #include <stdio.h>
-#endif FILE
+#endif /* FILE */
 #include <sys/time.h>
 #ifdef OW_I18N
 #endif /* OW_I18N */
diff -Nur xview-3.2p1.4.orig/lib/libxview/frame/fm_props.h xview-3.2p1.4/lib/libxview/frame/fm_props.h
--- xview-3.2p1.4.orig/lib/libxview/frame/fm_props.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/frame/fm_props.h	Mon Aug 19 22:48:01 2002
@@ -16,9 +16,9 @@
 #ifndef FILE
 #if !defined(SVR4) && !defined(__linux__)
 #undef NULL
-#endif SVR4
+#endif /* SVR4 */
 #include <stdio.h>
-#endif FILE
+#endif /* FILE */
 #include <sys/time.h>
 #include <xview/notify.h>
 #include <xview/rect.h>
@@ -45,7 +45,7 @@
 #include <pixrect/pixfont.h>
 #include <rasterfile.h>
 #include <pixrect/pr_io.h>
-#endif	pixrect_hs_DEFINED
+#endif	/* pixrect_hs_DEFINED */
 
 #include <xview/win_struct.h>	/* for WL_ links */
 #include <xview/win_input.h>
diff -Nur xview-3.2p1.4.orig/lib/libxview/frame/fm_xutil.h xview-3.2p1.4/lib/libxview/frame/fm_xutil.h
--- xview-3.2p1.4.orig/lib/libxview/frame/fm_xutil.h	Tue Jun 29 07:16:14 1993
+++ xview-3.2p1.4/lib/libxview/frame/fm_xutil.h	Mon Aug 19 22:48:01 2002
@@ -39,4 +39,4 @@
 /* frame_xutil.c */
 Xv_private void		frame_setwmhints();
 
-#endif frame_xutil_h_already_included
+#endif /* frame_xutil_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/frame/frame_base.h xview-3.2p1.4/lib/libxview/frame/frame_base.h
--- xview-3.2p1.4.orig/lib/libxview/frame/frame_base.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/frame/frame_base.h	Mon Aug 19 22:48:01 2002
@@ -20,9 +20,9 @@
 #ifndef FILE
 #if !defined(SVR4) && !defined(__linux__)
 #undef NULL
-#endif SVR4
+#endif /* SVR4 */
 #include <stdio.h>
-#endif FILE
+#endif /* FILE */
 #include <sys/time.h>
 #include <xview/notify.h>
 #include <xview/rect.h>
diff -Nur xview-3.2p1.4.orig/lib/libxview/frame/frame_cmd.h xview-3.2p1.4/lib/libxview/frame/frame_cmd.h
--- xview-3.2p1.4.orig/lib/libxview/frame/frame_cmd.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/frame/frame_cmd.h	Mon Aug 19 22:48:01 2002
@@ -20,9 +20,9 @@
 #ifndef FILE
 #if !defined(SVR4) && !defined(__linux__)
 #undef NULL
-#endif SVR4
+#endif /* SVR4 */
 #include <stdio.h>
-#endif FILE
+#endif /* FILE */
 #include <sys/time.h>
 #include <xview/notify.h>
 #include <xview/rect.h>
diff -Nur xview-3.2p1.4.orig/lib/libxview/frame/frame_help.h xview-3.2p1.4/lib/libxview/frame/frame_help.h
--- xview-3.2p1.4.orig/lib/libxview/frame/frame_help.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/frame/frame_help.h	Mon Aug 19 22:48:01 2002
@@ -20,9 +20,9 @@
 #ifndef FILE
 #if !defined(SVR4) && !defined(__linux__)
 #undef NULL
-#endif SVR4
+#endif /* SVR4 */
 #include <stdio.h>
-#endif FILE
+#endif /* FILE */
 #include <sys/time.h>
 #include <xview/notify.h>
 #include <xview/rect.h>
diff -Nur xview-3.2p1.4.orig/lib/libxview/help/help.h xview-3.2p1.4/lib/libxview/help/help.h
--- xview-3.2p1.4.orig/lib/libxview/help/help.h	Tue Jun 29 07:16:18 1993
+++ xview-3.2p1.4/lib/libxview/help/help.h	Mon Aug 19 22:48:01 2002
@@ -14,4 +14,4 @@
  */
 EXTERN_FUNCTION (int xv_help_show, (Xv_Window client_window, char *client_data, Event *client_event));
 
-#endif __xv_help_h
+#endif /* __xv_help_h */
diff -Nur xview-3.2p1.4.orig/lib/libxview/imcb/aux.h xview-3.2p1.4/lib/libxview/imcb/aux.h
--- xview-3.2p1.4.orig/lib/libxview/imcb/aux.h	Tue Jun 29 07:15:54 1993
+++ xview-3.2p1.4/lib/libxview/imcb/aux.h	Mon Aug 19 22:48:01 2002
@@ -85,4 +85,4 @@
 	int				f_height;
 }  AuxInfo;
 
-#endif		~aux_DEFINED
+#endif		/* ~aux_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/menu/om_impl.h xview-3.2p1.4/lib/libxview/menu/om_impl.h
--- xview-3.2p1.4.orig/lib/libxview/menu/om_impl.h	Fri May  2 14:21:46 1997
+++ xview-3.2p1.4/lib/libxview/menu/om_impl.h	Mon Aug 19 22:48:01 2002
@@ -22,7 +22,7 @@
 #ifdef OW_I18N
 #include <X11/Xlib.h>
 #include <X11/Xresource.h>
-#endif OW_I18N
+#endif /* OW_I18N */
 
 /***** Definitions *****/
 #ifndef TRUE
@@ -198,7 +198,7 @@
 		pin_window_header;
 #else
     char	*pin_window_header;
-#endif OW_I18N
+#endif /* OW_I18N */
     Xv_Window	window;			/* window containing menu items */
     Xv_Window	shadow_window;		/* gray shadow window */
 #ifdef OW_I18N
@@ -300,4 +300,4 @@
 }  Xv_menu_item_info;
 
 
-#endif _xview_walkmenu_impl_h_already_included
+#endif /* _xview_walkmenu_impl_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/menu/omi.h xview-3.2p1.4/lib/libxview/menu/omi.h
--- xview-3.2p1.4.orig/lib/libxview/menu/omi.h	Tue Jun 29 07:16:23 1993
+++ xview-3.2p1.4/lib/libxview/menu/omi.h	Mon Aug 19 22:48:01 2002
@@ -103,4 +103,4 @@
    extern void		 image_destroy();
    extern void		 image_render();
 
-#endif ~xview_image_DEFINED
+#endif /* ~xview_image_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/menu/omi_impl.h xview-3.2p1.4/lib/libxview/menu/omi_impl.h
--- xview-3.2p1.4.orig/lib/libxview/menu/omi_impl.h	Tue Jun 29 07:16:22 1993
+++ xview-3.2p1.4/lib/libxview/menu/omi_impl.h	Mon Aug 19 22:48:01 2002
@@ -15,9 +15,9 @@
 #ifndef pixrect_DEFINED
 #ifndef _TYPES_
 #include <sys/types.h>
-#endif _TYPES_
+#endif /* _TYPES_ */
 #include <pixrect/pixrect.h>
-#endif pixrect_DEFINED
+#endif /* pixrect_DEFINED */
 #include <xview/font.h>
 #include <xview/svrimage.h>
 #ifdef OW_I18N
@@ -87,4 +87,4 @@
 
 #define image_vector(x1,y1,x2,y2) \
 	(xv_vector(window,x1,y1,x2,y2,PIX_SET,0))
-#endif _xview_image_impl_h_already_included
+#endif /* _xview_image_impl_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/misc/bitmask.h xview-3.2p1.4/lib/libxview/misc/bitmask.h
--- xview-3.2p1.4.orig/lib/libxview/misc/bitmask.h	Tue Jun 29 07:16:28 1993
+++ xview-3.2p1.4/lib/libxview/misc/bitmask.h	Mon Aug 19 22:48:01 2002
@@ -32,6 +32,6 @@
 extern int xv_bitss_cmp_mask();
 extern Bitmask * xv_bitss_copy_mask();
 
-#endif bitmask_c
+#endif /* bitmask_c */
 
-#endif bitmask_h
+#endif /* bitmask_h */
diff -Nur xview-3.2p1.4.orig/lib/libxview/misc/gettext.h xview-3.2p1.4/lib/libxview/misc/gettext.h
--- xview-3.2p1.4.orig/lib/libxview/misc/gettext.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/misc/gettext.h	Mon Aug 19 22:48:01 2002
@@ -20,7 +20,7 @@
 #if defined(__linux__) && !defined(LC_MESSAGES) && defined(LC_RESPONSE)
 #define LC_MESSAGES LC_RESPONSE
 #endif
-#endif OS_HAS_LOCALE
+#endif /* OS_HAS_LOCALE */
 #include <stdio.h>
 #include <sys/types.h>
 #include <sys/param.h>
diff -Nur xview-3.2p1.4.orig/lib/libxview/misc/hashfn.h xview-3.2p1.4/lib/libxview/misc/hashfn.h
--- xview-3.2p1.4.orig/lib/libxview/misc/hashfn.h	Tue Jun 29 07:16:31 1993
+++ xview-3.2p1.4/lib/libxview/misc/hashfn.h	Mon Aug 19 22:48:01 2002
@@ -49,5 +49,5 @@
 extern caddr_t /* key pointer */ hashfn_first_key();
 extern caddr_t /* key pointer */ hashfn_next_key();
 
-#endif hashfn_c
-#endif hashfn_h
+#endif /* hashfn_c */
+#endif /* hashfn_h */
diff -Nur xview-3.2p1.4.orig/lib/libxview/notice/noticeimpl.h xview-3.2p1.4/lib/libxview/notice/noticeimpl.h
--- xview-3.2p1.4.orig/lib/libxview/notice/noticeimpl.h	Tue Jun 29 07:16:37 1993
+++ xview-3.2p1.4/lib/libxview/notice/noticeimpl.h	Mon Aug 19 22:48:01 2002
@@ -204,4 +204,4 @@
 Pkg_private int		notice_context_key;
 Pkg_private Defaults_pairs bell_types[];
 
-#endif notice_impl_h_already_defined
+#endif /* notice_impl_h_already_defined */
diff -Nur xview-3.2p1.4.orig/lib/libxview/notify/linux_select.h xview-3.2p1.4/lib/libxview/notify/linux_select.h
--- xview-3.2p1.4.orig/lib/libxview/notify/linux_select.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/notify/linux_select.h	Mon Aug 19 22:48:01 2002
@@ -17,4 +17,4 @@
 int sys_read(int fildes, char * buf, off_t cnt);
 #endif
 
-#endif __LINUX_SELECT_H
+#endif /* __LINUX_SELECT_H */
diff -Nur xview-3.2p1.4.orig/lib/libxview/notify/ndet.h xview-3.2p1.4/lib/libxview/notify/ndet.h
--- xview-3.2p1.4.orig/lib/libxview/notify/ndet.h	Tue Jun 29 07:18:12 1993
+++ xview-3.2p1.4/lib/libxview/notify/ndet.h	Mon Aug 19 22:48:01 2002
@@ -380,5 +380,5 @@
 immediate client events.
 */
 
-#endif	NDET_DEFINED
+#endif	/* NDET_DEFINED */
 
diff -Nur xview-3.2p1.4.orig/lib/libxview/notify/ndis.h xview-3.2p1.4/lib/libxview/notify/ndis.h
--- xview-3.2p1.4.orig/lib/libxview/notify/ndis.h	Tue Jun 29 07:18:14 1993
+++ xview-3.2p1.4/lib/libxview/notify/ndis.h	Mon Aug 19 22:48:01 2002
@@ -125,5 +125,5 @@
 					    int use_data) */
 void	ndis_flush_wait3();		/* Flush all conditions for pid
 					   (Notify_client client, int pid) */
-#endif	NDIS_DEFINED
+#endif	/* NDIS_DEFINED */
 
diff -Nur xview-3.2p1.4.orig/lib/libxview/notify/nint.h xview-3.2p1.4/lib/libxview/notify/nint.h
--- xview-3.2p1.4.orig/lib/libxview/notify/nint.h	Tue Jun 29 07:18:05 1993
+++ xview-3.2p1.4/lib/libxview/notify/nint.h	Mon Aug 19 22:48:01 2002
@@ -127,5 +127,5 @@
 						   nint_pop_callout exceptdoes
 						   assumes in critical section*/
 
-#endif	NINT_DEFINED
+#endif	/* NINT_DEFINED */
 
diff -Nur xview-3.2p1.4.orig/lib/libxview/notify/ntfy.h xview-3.2p1.4/lib/libxview/notify/ntfy.h
--- xview-3.2p1.4.orig/lib/libxview/notify/ntfy.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/notify/ntfy.h	Mon Aug 19 22:48:01 2002
@@ -169,7 +169,7 @@
 	int	pid;			/* Process waiting for */
 #if !defined(SVR4) && !defined(__linux__)
 	union	wait status;		/* Return value from wait3 */
-#else SVR4
+#else /* SVR4 */
 	int 	status;		/* Return value from wait3 */
 #endif
 	struct	rusage rusage;		/* Return value from wait3 */
@@ -489,9 +489,9 @@
 #ifdef	NTFY_DEBUG
 #define	ntfy_set_errno(err)	ntfy_set_errno_debug((err))
 void	ntfy_set_errno_debug();
-#else	NTFY_DEBUG
+#else	/* NTFY_DEBUG */
 #define	ntfy_set_errno(err)	notify_errno = err
-#endif	NTFY_DEBUG
+#endif	/* NTFY_DEBUG */
 /*
  * Ntfy_set_warning is for setting notify_errno when you don't usually want
  * to generate a message; the caller may be using the call in a valid manner
@@ -502,21 +502,21 @@
 #ifdef	NTFY_DEBUG
 #define	ntfy_set_warning(err) ntfy_set_warning_debug((err))
 void	ntfy_set_warning_debug();
-#else	NTFY_DEBUG
+#else	/* NTFY_DEBUG */
 #define	ntfy_set_warning(err) notify_errno = err
-#endif	NTFY_DEBUG
+#endif	/* NTFY_DEBUG */
 
 #ifdef	NTFY_DEBUG
 #define	ntfy_assert(bool, code)  if (!(bool)) ntfy_assert_debug(code)
 void	ntfy_assert_debug();
-#else	NTFY_DEBUG
+#else	/* NTFY_DEBUG */
 #define	ntfy_assert(bool, code)	{}
-#endif	NTFY_DEBUG
+#endif	/* NTFY_DEBUG */
 
 void	ntfy_fatal_error();
 
 #define	pkg_private	extern
 #define	pkg_private_data
 
-#endif	NTFY_DEFINED
+#endif	/* NTFY_DEFINED */
 
diff -Nur xview-3.2p1.4.orig/lib/libxview/notify/ultrix_cpt.h xview-3.2p1.4/lib/libxview/notify/ultrix_cpt.h
--- xview-3.2p1.4.orig/lib/libxview/notify/ultrix_cpt.h	Tue Jun 29 07:18:07 1993
+++ xview-3.2p1.4/lib/libxview/notify/ultrix_cpt.h	Mon Aug 19 22:48:01 2002
@@ -49,6 +49,6 @@
 #define FD_ISSET(n, p)  ((p)->fds_bits[0] & (1 << ((n) % NFDBITS)))
 #define FD_ZERO(p)      ((p)->fds_bits[0] = 0)
 
-#endif OLD_BSD_FDSETS
+#endif /* OLD_BSD_FDSETS */
 
-#endif ~xview_ultrix_compat_DEFINED
+#endif /* ~xview_ultrix_compat_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/openwin/ow_impl.h xview-3.2p1.4/lib/libxview/openwin/ow_impl.h
--- xview-3.2p1.4.orig/lib/libxview/openwin/ow_impl.h	Tue Jun 29 07:16:44 1993
+++ xview-3.2p1.4/lib/libxview/openwin/ow_impl.h	Mon Aug 19 22:48:01 2002
@@ -91,7 +91,7 @@
 #ifdef SELECTABLE_VIEWS
 	Seln_client	seln_client;		/* selection svc client id */
 	Openwin_view_info	*seln_view;	/* selected view, if any */
-#endif SELECTABLE_VIEWS
+#endif /* SELECTABLE_VIEWS */
 	struct {
 	    BOOLEAN_FIELD(auto_clear);
 	    BOOLEAN_FIELD(adjust_vertical);
@@ -153,7 +153,7 @@
 extern Xv_opaque openwin_seln_reply();
 extern void	 openwin_select();
 extern void	 openwin_select_view();
-#endif SELECTABLE_VIEWS
+#endif /* SELECTABLE_VIEWS */
 
 /* openwin_view.c */
 extern	void				openwin_create_initial_view();
@@ -165,4 +165,4 @@
 extern	int				openwin_fill_view_gap();
 extern	void				openwin_copy_scrollbar();
 
-#endif	__openwin_impl_h
+#endif	/* __openwin_impl_h */
diff -Nur xview-3.2p1.4.orig/lib/libxview/panel/panel_impl.h xview-3.2p1.4/lib/libxview/panel/panel_impl.h
--- xview-3.2p1.4.orig/lib/libxview/panel/panel_impl.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/panel/panel_impl.h	Mon Aug 19 22:48:01 2002
@@ -12,9 +12,9 @@
 #ifndef FILE
 #if !defined(SVR4) && !defined(__linux__)
 #undef NULL
-#endif SVR4
+#endif /* SVR4 */
 #include <stdio.h>
-#endif FILE
+#endif /* FILE */
 #include <sys/types.h>
 #include <X11/Xlib.h>
 #include <xview_private/i18n_impl.h>
diff -Nur xview-3.2p1.4.orig/lib/libxview/scrollbar/sb_impl.h xview-3.2p1.4/lib/libxview/scrollbar/sb_impl.h
--- xview-3.2p1.4.orig/lib/libxview/scrollbar/sb_impl.h	Tue Jun 29 07:17:09 1993
+++ xview-3.2p1.4/lib/libxview/scrollbar/sb_impl.h	Mon Aug 19 22:48:01 2002
@@ -104,9 +104,9 @@
 /* dimensions for extralarge size scrollbar */
 #ifndef SVR4
 #define SCROLLBAR_XLARGE_THICKNESS		29
-#else SVR4
+#else /* SVR4 */
 #define SCROLLBAR_XLARGE_THICKNESS		27
-#endif SVR4
+#endif /* SVR4 */
 #define	SB_XLARGE_MARGIN			 4
 #define	SB_XLARGE_MARKER_HEIGHT			 9
 #define SB_XLARGE_ELEVATOR_BOX_HEIGHT	 	21
@@ -251,4 +251,4 @@
 Pkg_private void scrollbar_join_view_from_menu();
 Pkg_private void scrollbar_init_delay_values();
 
-#endif	__scrollbar_impl_h
+#endif	/* __scrollbar_impl_h */
diff -Nur xview-3.2p1.4.orig/lib/libxview/sel/sel_compat.h xview-3.2p1.4/lib/libxview/sel/sel_compat.h
--- xview-3.2p1.4.orig/lib/libxview/sel/sel_compat.h	Tue Jun 29 07:15:22 1993
+++ xview-3.2p1.4/lib/libxview/sel/sel_compat.h	Mon Aug 19 22:48:01 2002
@@ -43,4 +43,4 @@
 #define seln_report_event(seln_client, event)	\
 		selection_report_event(xv_default_server, seln_client, event)
 
-#endif ~xview_selection_compat_DEFINED
+#endif /* ~xview_selection_compat_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/sel/seln_impl.h xview-3.2p1.4/lib/libxview/sel/seln_impl.h
--- xview-3.2p1.4.orig/lib/libxview/sel/seln_impl.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/sel/seln_impl.h	Mon Aug 19 22:48:01 2002
@@ -13,9 +13,9 @@
 #ifndef FILE
 #if !defined(SVR4) && !defined(__linux__)
 #undef NULL
-#endif SVR4
+#endif /* SVR4 */
 #include <stdio.h>
-#endif FILE
+#endif /* FILE */
 #include <sys/time.h>
 #include <sys/types.h>
 #include <netdb.h>
@@ -97,7 +97,7 @@
     Time		timestamp;
     unsigned char	format;
 } Seln_agent_context;
-#endif OW_I18N
+#endif /* OW_I18N */
 
 typedef struct {
     long		offset;
@@ -203,7 +203,7 @@
 							selection.first_wc
 #define LAST_WC(selection)	"_SUN_SELN_LAST_WC", SELN_REQ_LAST_WC, \
 							selection.last_wc
-#endif OW_I18N
+#endif /* OW_I18N */
 
 #define IS_READONLY(selection)	"_SUN_SELN_IS_READONLY", SELN_REQ_IS_READONLY, \
 							   selection.is_readonly
diff -Nur xview-3.2p1.4.orig/lib/libxview/selection/sel_impl.h xview-3.2p1.4/lib/libxview/selection/sel_impl.h
--- xview-3.2p1.4.orig/lib/libxview/selection/sel_impl.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/selection/sel_impl.h	Mon Aug 19 22:48:01 2002
@@ -112,7 +112,7 @@
     Atom         integer;
 #ifdef OW_I18N
     Atom	 ctext;
-#endif OW_I18N
+#endif /* OW_I18N */
 } Sel_atom_list;
 
 
diff -Nur xview-3.2p1.4.orig/lib/libxview/server_image/svrim_impl.h xview-3.2p1.4/lib/libxview/server_image/svrim_impl.h
--- xview-3.2p1.4.orig/lib/libxview/server_image/svrim_impl.h	Tue Jun 29 07:15:33 1993
+++ xview-3.2p1.4/lib/libxview/server_image/svrim_impl.h	Mon Aug 19 22:48:01 2002
@@ -40,4 +40,4 @@
 Xv_private GC       xv_find_proper_gc();
 Xv_private Pixrect  *xv_mem_create();
 
-#endif  _xview_server_image_impl_h_already_included
+#endif  /* _xview_server_image_impl_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/textsw/primal.h xview-3.2p1.4/lib/libxview/textsw/primal.h
--- xview-3.2p1.4.orig/lib/libxview/textsw/primal.h	Tue Jun 29 07:17:29 1993
+++ xview-3.2p1.4/lib/libxview/textsw/primal.h	Mon Aug 19 22:48:01 2002
@@ -35,14 +35,14 @@
 #ifndef SVR4
 #define IDENTITY(formal)\
 	formal
-#endif SVR4
+#endif /* SVR4 */
 #ifndef SVR4
 #define CONCATENATE(formal1,formal2)\
 	IDENTITY(formal1)formal2
-#else SVR4
+#else /* SVR4 */
 #define CONCATENATE(formal1,formal2)\
         formal1##formal2
-#endif SVR4
+#endif /* SVR4 */
 
 #define	pkg_private	extern
 
diff -Nur xview-3.2p1.4.orig/lib/libxview/ttysw/term_impl.h xview-3.2p1.4/lib/libxview/ttysw/term_impl.h
--- xview-3.2p1.4.orig/lib/libxview/ttysw/term_impl.h	Tue Jun 29 07:17:23 1993
+++ xview-3.2p1.4/lib/libxview/ttysw/term_impl.h	Mon Aug 19 22:48:01 2002
@@ -238,7 +238,7 @@
 #define	textsw_add_mark_i18n		textsw_add_mark_wc
 #define	textsw_erase_i18n		textsw_erase_wcs
 #define	textsw_delete_i18n		textsw_delete_wcs
-#else /OW_I18N */
+#else /* OW_I18N */
 #define	TEXTSW_LENGTH_I18N		TEXTSW_LENGTH
 #define	TEXTSW_CONTENTS_I18N		TEXTSW_CONTENTS
 #define	textsw_replace_i18n		textsw_replace_bytes
@@ -251,4 +251,4 @@
 #define	textsw_delete_i18n		textsw_delete
 #endif /* OW_I18N */
 
-#endif _xview_private_termsw_impl_h_already_included
+#endif /* _xview_private_termsw_impl_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/ttysw/tty_impl.h xview-3.2p1.4/lib/libxview/ttysw/tty_impl.h
--- xview-3.2p1.4.orig/lib/libxview/ttysw/tty_impl.h	Mon Aug 19 21:43:42 2002
+++ xview-3.2p1.4/lib/libxview/ttysw/tty_impl.h	Mon Aug 19 22:48:01 2002
@@ -463,6 +463,6 @@
 ttysw_input(Tty ttysw_public, char *addr, int len);
 
 /* Add string to the input queue. */
-#endif	cplus
+#endif	/* cplus */
 
-#endif _xview_private_ttysw_impl_h_already_included
+#endif /* _xview_private_ttysw_impl_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/libxview/win/win_ascii.h xview-3.2p1.4/lib/libxview/win/win_ascii.h
--- xview-3.2p1.4.orig/lib/libxview/win/win_ascii.h	Tue Jun 29 07:15:45 1993
+++ xview-3.2p1.4/lib/libxview/win/win_ascii.h	Mon Aug 19 22:48:01 2002
@@ -1872,4 +1872,4 @@
 };
 
 
-#endif win_ascii_data_DEFINED
+#endif /* win_ascii_data_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/win/win_keymap.h xview-3.2p1.4/lib/libxview/win/win_keymap.h
--- xview-3.2p1.4.orig/lib/libxview/win/win_keymap.h	Tue Jun 29 07:15:39 1993
+++ xview-3.2p1.4/lib/libxview/win/win_keymap.h	Mon Aug 19 22:48:01 2002
@@ -53,4 +53,4 @@
 
 
 
-#endif	win_keymap_DEFINED
+#endif	/* win_keymap_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/win/win_kmdata.h xview-3.2p1.4/lib/libxview/win/win_kmdata.h
--- xview-3.2p1.4.orig/lib/libxview/win/win_kmdata.h	Tue Jun 29 07:15:37 1993
+++ xview-3.2p1.4/lib/libxview/win/win_kmdata.h	Mon Aug 19 22:48:01 2002
@@ -48,7 +48,7 @@
 #else    
         /* BUG: Only in R4. */
         XK_Scroll_Lock,                 		/* XK_Scroll_Lock */
-#endif XK_Scroll_Lock
+#endif /* XK_Scroll_Lock */
 	NULL, NULL, NULL, NULL, NULL, NULL,
 
 	XK_Escape,
@@ -211,4 +211,4 @@
 };
 
 
-#endif win_keymap_data_DEFINED
+#endif /* win_keymap_data_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/libxview/wmgr/wmgr_decor.h xview-3.2p1.4/lib/libxview/wmgr/wmgr_decor.h
--- xview-3.2p1.4.orig/lib/libxview/wmgr/wmgr_decor.h	Tue Jun 29 07:17:25 1993
+++ xview-3.2p1.4/lib/libxview/wmgr/wmgr_decor.h	Mon Aug 19 22:48:01 2002
@@ -59,13 +59,13 @@
 /* value for pushpin_initial_state */
 #ifndef WMPushpinIsOut
 #define WMPushpinIsOut	0
-#endif WMPushpinIsOut
+#endif /* WMPushpinIsOut */
 #ifndef WMPushpinIsIn
 #define WMPushpinIsIn	1
-#endif WMPushpinIsIn
+#endif /* WMPushpinIsIn */
 
 /* value for WM_WINDOW_BUSY property */
 #define WMWindowNotBusy	0
 #define WMWindowIsBusy	1
 
-#endif _wmgr_decor_h_already_included
+#endif /* _wmgr_decor_h_already_included */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/bw2var.h xview-3.2p1.4/lib/pixrect/bw2var.h
--- xview-3.2p1.4.orig/lib/pixrect/bw2var.h	Tue Jun 29 07:18:36 1993
+++ xview-3.2p1.4/lib/pixrect/bw2var.h	Mon Aug 19 22:48:01 2002
@@ -37,4 +37,4 @@
 int	bw2_destroy();
 #endif
 
-#endif	bw2var_DEFINED
+#endif	/* bw2var_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/cg12_var.h xview-3.2p1.4/lib/pixrect/cg12_var.h
--- xview-3.2p1.4.orig/lib/pixrect/cg12_var.h	Tue Jun 29 07:18:36 1993
+++ xview-3.2p1.4/lib/pixrect/cg12_var.h	Mon Aug 19 22:48:01 2002
@@ -13,17 +13,17 @@
 #include <sun/fbio.h>
 #else
 #include <sys/fbio.h>
-#endif SVR4
+#endif /* SVR4 */
 #include <pixrect/pixrect.h>
 #ifndef SVR4
 #include <sbusdev/cg12reg.h>
-#endif SVR4
+#endif /* SVR4 */
 #include <pixrect/memvar.h>
 #ifndef SVR4
 #include <sunwindow/cms.h>		/* colormapseg */
 #else
 #include <sys/cms.h>
-#endif SVR4
+#endif /* SVR4 */
 
 #define	CG12_NFBS		6	/* number of frame buffers in a CG12 */
 
diff -Nur xview-3.2p1.4.orig/lib/pixrect/cg2reg.h xview-3.2p1.4/lib/pixrect/cg2reg.h
--- xview-3.2p1.4.orig/lib/pixrect/cg2reg.h	Tue Jun 29 07:18:36 1993
+++ xview-3.2p1.4/lib/pixrect/cg2reg.h	Mon Aug 19 22:48:01 2002
@@ -310,4 +310,4 @@
 
 #define	cg2_touch(a)	((a)=0)
 
-#endif	cg2reg_DEFINED
+#endif	/* cg2reg_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/cg2var.h xview-3.2p1.4/lib/pixrect/cg2var.h
--- xview-3.2p1.4.orig/lib/pixrect/cg2var.h	Tue Jun 29 07:18:36 1993
+++ xview-3.2p1.4/lib/pixrect/cg2var.h	Mon Aug 19 22:48:01 2002
@@ -83,6 +83,6 @@
 Pixrect *cg2_region();
 int	cg2_getcolormap();
 int	cg2_getattributes();
-#endif !KERNEL
+#endif /* !KERNEL */
 
-#endif	cg2var_DEFINED
+#endif	/* cg2var_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/cg3var.h xview-3.2p1.4/lib/pixrect/cg3var.h
--- xview-3.2p1.4.orig/lib/pixrect/cg3var.h	Tue Jun 29 07:18:37 1993
+++ xview-3.2p1.4/lib/pixrect/cg3var.h	Mon Aug 19 22:48:01 2002
@@ -45,6 +45,6 @@
 Pixrect *cg3_region();
 int cg3_putcolormap();
 int cg3_getcolormap();
-#endif !KERNEL
+#endif /* !KERNEL */
 
-#endif	!cg3var_DEFINED
+#endif	/* !cg3var_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/cg4var.h xview-3.2p1.4/lib/pixrect/cg4var.h
--- xview-3.2p1.4.orig/lib/pixrect/cg4var.h	Tue Jun 29 07:18:37 1993
+++ xview-3.2p1.4/lib/pixrect/cg4var.h	Mon Aug 19 22:48:01 2002
@@ -54,6 +54,6 @@
 Pixrect *cg4_region();
 int	cg4_getcolormap();
 int	cg4_getattributes();
-#endif !KERNEL
+#endif /* !KERNEL */
 
-#endif	!cg4var_DEFINED
+#endif	/* !cg4var_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/cg8var.h xview-3.2p1.4/lib/pixrect/cg8var.h
--- xview-3.2p1.4.orig/lib/pixrect/cg8var.h	Tue Jun 29 07:18:37 1993
+++ xview-3.2p1.4/lib/pixrect/cg8var.h	Mon Aug 19 22:48:01 2002
@@ -14,7 +14,7 @@
 #include <sunwindow/cms.h>
 #else
 #include <sys/cms.h>
-#endif SVR4
+#endif /* SVR4 */
 
 /* FBIOSATTR device specific array indices, copied from cg4var.h */
 #define	FB_ATTR_CG8_SETOWNER_CMD	0	/* 1 indicates PID is valid */
@@ -90,6 +90,6 @@
 int		cg8_put();
 int		cg8_rop();
 
-#endif	!KERNEL
+#endif	/* !KERNEL */
 
-#endif cg8var_DEFINED
+#endif /* cg8var_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/cg9var.h xview-3.2p1.4/lib/pixrect/cg9var.h
--- xview-3.2p1.4.orig/lib/pixrect/cg9var.h	Tue Jun 29 07:18:37 1993
+++ xview-3.2p1.4/lib/pixrect/cg9var.h	Mon Aug 19 22:48:01 2002
@@ -13,13 +13,13 @@
 
 #ifndef SVR4
 #include	<sundev/cg9reg.h>
-#endif SVR4
+#endif /* SVR4 */
 #include	<pixrect/memvar.h>
 #ifndef SVR4
 #include	<sunwindow/cms.h>	/* colormapseg */
 #else
 #include <sys/cms.h>
-#endif SVR4
+#endif /* SVR4 */
 
 /* description of single CG9 frame buffer */
 
@@ -92,6 +92,6 @@
 int		cg9_get();
 int		cg9_put();
 
-#endif	!KERNEL
+#endif	/* !KERNEL */
 
-#endif	cg9var_DEFINED
+#endif	/* cg9var_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/gp1cmds.h xview-3.2p1.4/lib/pixrect/gp1cmds.h
--- xview-3.2p1.4.orig/lib/pixrect/gp1cmds.h	Tue Jun 29 07:18:37 1993
+++ xview-3.2p1.4/lib/pixrect/gp1cmds.h	Mon Aug 19 22:48:01 2002
@@ -374,7 +374,7 @@
 #define	GP1_GET_I(p, a)	((a) = * (int *) (p), \
 				(p) += sizeof (int) / sizeof *(p))
 
-#else  mc68000
+#else  /* mc68000 */
 
 #define	GP1_PUT_F(p, a)	(((short *) (p))[0] = ((short *)&(a))[0], \
 				((short *) (p))[1] = ((short *) &(a))[1] , \
@@ -392,7 +392,7 @@
 				((short *) &(a))[1] = ((short *) (p))[1], \
 				(p) += sizeof (int) / sizeof *(p))
 
-#endif mc68000
+#endif /* mc68000 */
 
 
 /* The names of these macros have been changed to avoid 
@@ -403,4 +403,4 @@
 #define gp1_pw_offset_X(p) (gp1_d((p)->pw_clipdata->pwcd_prmulti)->cgpr_offset.x)
 #define gp1_pw_offset_Y(p) (gp1_d((p)->pw_clipdata->pwcd_prmulti)->cgpr_offset.y)
 
-#endif	gp1cmds_DEFINED
+#endif	/* gp1cmds_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/gp1var.h xview-3.2p1.4/lib/pixrect/gp1var.h
--- xview-3.2p1.4.orig/lib/pixrect/gp1var.h	Tue Jun 29 07:18:38 1993
+++ xview-3.2p1.4/lib/pixrect/gp1var.h	Mon Aug 19 22:48:01 2002
@@ -15,7 +15,7 @@
 #include <sun/fbio.h>
 #else
 #include <sys/fbio.h>
-#endif SVR4
+#endif /* SVR4 */
 #include <pixrect/cg2var.h>
 #include <pixrect/cg9var.h>
 
@@ -106,10 +106,10 @@
 int gp1_stencil();
 int gp1_polypoint();
 
-#endif !KERNEL
+#endif /* !KERNEL */
 
 #define GP1IO_SATTR _IOW(G, 101, struct fbgattr)
 #define GP1IO_SCMAP _IO(G, 102)
 #define GP_SHMEMSIZE	5
 
-#endif	gp1var_DEFINED
+#endif	/* gp1var_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/mem32_var.h xview-3.2p1.4/lib/pixrect/mem32_var.h
--- xview-3.2p1.4.orig/lib/pixrect/mem32_var.h	Tue Jun 29 07:18:38 1993
+++ xview-3.2p1.4/lib/pixrect/mem32_var.h	Mon Aug 19 22:48:01 2002
@@ -13,7 +13,7 @@
 #include	<sunwindow/cms.h>	/* colormapseg */
 #else
 #include <sys/cms.h>
-#endif SVR4
+#endif /* SVR4 */
 
 #define         MEM32_8BIT_CMAPSIZE     256
 
diff -Nur xview-3.2p1.4.orig/lib/pixrect/mem_rop_impl_util.h xview-3.2p1.4/lib/pixrect/mem_rop_impl_util.h
--- xview-3.2p1.4.orig/lib/pixrect/mem_rop_impl_util.h	Tue Jun 29 07:18:39 1993
+++ xview-3.2p1.4/lib/pixrect/mem_rop_impl_util.h	Mon Aug 19 22:48:01 2002
@@ -58,4 +58,4 @@
 #define UFAST IFKERNEL(SLOW,FAST)
 #define UFASTT IFKERNEL(SLOWT,FASTT)
 
-#endif	mem_rop_impl_util_DEFINED
+#endif	/* mem_rop_impl_util_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/memreg.h xview-3.2p1.4/lib/pixrect/memreg.h
--- xview-3.2p1.4.orig/lib/pixrect/memreg.h	Tue Jun 29 07:18:38 1993
+++ xview-3.2p1.4/lib/pixrect/memreg.h	Mon Aug 19 22:48:01 2002
@@ -61,4 +61,4 @@
 #define	mrc_lmask(x)	(0xffff0000 >> (x))
 #define	mrc_rmask(x)	(0x7fff >> (x))
 
-#endif	memreg_DEFINED
+#endif	/* memreg_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/memvar.h xview-3.2p1.4/lib/pixrect/memvar.h
--- xview-3.2p1.4.orig/lib/pixrect/memvar.h	Tue Jun 29 07:18:33 1993
+++ xview-3.2p1.4/lib/pixrect/memvar.h	Mon Aug 19 22:48:01 2002
@@ -48,7 +48,7 @@
 #ifdef i386
 #define MP_I386		8	/* Pixrect is for 386 architecture */
 #define MP_STATIC       16	/* Pixrect is a static pixrect */
-#endif i386
+#endif /* i386 */
 #define	MP_FONT		32	/* Pixrect is a part of a Pixfont */
 				/* (hint to pr_batchrop) */
 
@@ -144,18 +144,18 @@
 int	mem_put();
 int	mem_vector();
 Pixrect *mem_region();
-#endif KERNEL
+#endif /* KERNEL */
 int	mem_putcolormap();
 int	mem_putattributes();
 #ifndef KERNEL
 int	mem_getcolormap();
 int	mem_getattributes();
-#endif KERNEL
+#endif /* KERNEL */
 
 #else /* NEWPIXMEM */
 #define	MP_NOTMPR(pr)	(0)
 extern struct pixrectops mem_ops;
 int	mem_destroy();
-#endif NEWPIXMEM
+#endif /* NEWPIXMEM */
 
-#endif	!memvar_DEFINED
+#endif	/* !memvar_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pixfont.h xview-3.2p1.4/lib/pixrect/pixfont.h
--- xview-3.2p1.4.orig/lib/pixrect/pixfont.h	Tue Jun 29 07:18:33 1993
+++ xview-3.2p1.4/lib/pixrect/pixfont.h	Mon Aug 19 22:48:01 2002
@@ -43,7 +43,7 @@
 
 #define	prs_ttext(prpos, op, pf, str) \
 	pr_ttext((prpos).pr, (prpos).pos.x, (prpos).pos.y, pf, str)
-#endif lint
+#endif /* lint */
 
 Pixfont	*pf_open();
 Pixfont	*pf_open_private();
@@ -53,4 +53,4 @@
 
 #define	PIXFONT	Pixfont
 
-#endif	pixfont_DEFINED
+#endif	/* pixfont_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pixrect.h xview-3.2p1.4/lib/pixrect/pixrect.h
--- xview-3.2p1.4.orig/lib/pixrect/pixrect.h	Tue Jun 29 07:18:33 1993
+++ xview-3.2p1.4/lib/pixrect/pixrect.h	Mon Aug 19 22:48:01 2002
@@ -101,7 +101,7 @@
  * Takes device file name.  This is how a screen pixrect is created.
  */
 extern Pixrect *pr_open();
-#endif !KERNEL
+#endif /* !KERNEL */
 
 /*
  * Pixrect ops vector, used by pr_ macros below to call the appropriate
@@ -118,18 +118,18 @@
 	int	(*pro_put)();
 	int	(*pro_vector)();
 	Pixrect * (*pro_region)();
-#endif !KERNEL
+#endif /* !KERNEL */
 	int	(*pro_putcolormap)();
 #ifndef KERNEL
 	int	(*pro_getcolormap)();
-#endif !KERNEL
+#endif /* !KERNEL */
 	int	(*pro_putattributes)();
 #ifndef KERNEL
 	int	(*pro_getattributes)();
-#endif !KERNEL
+#endif /* !KERNEL */
 #ifdef KERNEL
 	int	(*pro_nop)();		/* place holder */
-#endif KERNEL
+#endif /* KERNEL */
 };
 
 #if !defined(lint) || defined(KERNEL)
@@ -169,9 +169,9 @@
 		(red), (grn), (blu))
 #define	pr_getattributes(pr, planes) \
 	(*(pr)->pr_ops->pro_getattributes)((pr), (planes))
-#endif !KERNEL
+#endif /* !KERNEL */
 
-#else !lint || KERNEL
+#else /* !lint || KERNEL */
 
 extern pr_rop();
 extern pr_stencil();
@@ -186,7 +186,7 @@
 extern pr_putattributes();
 extern pr_getattributes();
 
-#endif lint
+#endif /* lint */
 
 /*
  * Several of the above operations return a common, distinguished value when
@@ -277,7 +277,7 @@
 #define	pr_line(pr, x0, y0, x1, y1, brush, tex, op) \
 	pro_line((pr), (x0), (y0), (x1), (y1), (brush), (tex), (op), 0) 
 
-#else !lint || KERNEL
+#else /* !lint || KERNEL */
 
 extern prs_rop();
 extern prs_stencil();
@@ -296,7 +296,7 @@
 extern pr_close();
 extern pr_line();
 
-#endif !lint || KERNEL
+#endif /* !lint || KERNEL */
 
 
 
@@ -329,7 +329,7 @@
 #define	pr_getlut(pr, ind, cnt, red, grn, blu) \
 	(*(pr)->pr_ops->pro_getcolormap)((pr), PR_FORCE_UPDATE | (ind), \
 		(cnt), (red), (grn), (blu))
-#endif !KERNEL
+#endif /* !KERNEL */
 
 
 #endif	/* !_pixrect_pixrect_h */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pixrect_hs.h xview-3.2p1.4/lib/pixrect/pixrect_hs.h
--- xview-3.2p1.4.orig/lib/pixrect/pixrect_hs.h	Tue Jun 29 07:18:34 1993
+++ xview-3.2p1.4/lib/pixrect/pixrect_hs.h	Mon Aug 19 22:48:01 2002
@@ -18,7 +18,7 @@
 #include <pixrect/pr_dblbuf.h>
 #include <pixrect/pr_line.h>
 #include <pixrect/pr_planegroups.h>
-#endif NEWPIXMEM
+#endif /* NEWPIXMEM */
 
 #include <pixrect/pr_util.h>
 
@@ -32,7 +32,7 @@
 #include <pixrect/cg8var.h>
 #include <pixrect/cg9var.h>
 #include <pixrect/cg12_var.h>
-#endif NEWPIXMEM
+#endif /* NEWPIXMEM */
 
 #include <pixrect/memvar.h>
 
@@ -40,9 +40,9 @@
 #include <pixrect/mem32_var.h>
 
 #include <pixrect/pixfont.h>
-#endif NEWPIXMEM
+#endif /* NEWPIXMEM */
 
 #include <rasterfile.h>
 #include <pixrect/pr_io.h>
 
-#endif	pixrect_hs_DEFINED
+#endif	/* pixrect_hs_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pr_dblbuf.h xview-3.2p1.4/lib/pixrect/pr_dblbuf.h
--- xview-3.2p1.4.orig/lib/pixrect/pr_dblbuf.h	Tue Jun 29 07:18:34 1993
+++ xview-3.2p1.4/lib/pixrect/pr_dblbuf.h	Mon Aug 19 22:48:01 2002
@@ -33,4 +33,4 @@
 #define PR_DBL_NONE	5
     /* can only be returned by pr_dbl_get with PR_DBL_WRITE attribute. */
 
-#endif pr_dblbuf_DEFINED
+#endif /* pr_dblbuf_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pr_impl_make.h xview-3.2p1.4/lib/pixrect/pr_impl_make.h
--- xview-3.2p1.4.orig/lib/pixrect/pr_impl_make.h	Tue Jun 29 07:18:35 1993
+++ xview-3.2p1.4/lib/pixrect/pr_impl_make.h	Mon Aug 19 22:48:01 2002
@@ -25,6 +25,6 @@
 #ifndef KERNEL
 Pixrect *pr_makefromfd();
 Pixrect *pr_makefromfd_2();
-#endif !KERNEL
+#endif /* !KERNEL */
 
-#endif	pr_impl_make_DEFINED
+#endif	/* pr_impl_make_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pr_impl_util.h xview-3.2p1.4/lib/pixrect/pr_impl_util.h
--- xview-3.2p1.4.orig/lib/pixrect/pr_impl_util.h	Tue Jun 29 07:18:38 1993
+++ xview-3.2p1.4/lib/pixrect/pr_impl_util.h	Mon Aug 19 22:48:01 2002
@@ -55,14 +55,14 @@
 #define	_ONE_	(!_ZERO_)
 int _loop;	/* "_loop redefinition hides earlier one" */
 
-#else lint
+#else /* lint */
 
 #define	IFLINT	IFFALSE
 
 #define	_ZERO_ 0
 #define	_ONE_ 1
 
-#endif lint
+#endif /* lint */
 
 /*
  * portability aids
@@ -77,7 +77,7 @@
 
 #define	LOOP_DECR(var)	(--(var) != -1)
 
-#else mc68000
+#else /* mc68000 */
 
 #define IF68000	IFFALSE
 
@@ -87,7 +87,7 @@
 
 #define	LOOP_DECR(var)	(--(var) >= 0)
 
-#endif mc68000
+#endif /* mc68000 */
 
 #ifdef sparc
 #define	IFSPARC	IFTRUE
@@ -124,13 +124,13 @@
 #if defined(sun) && !defined(SUNOS)
 #ifdef _sys_types_h
 #define	SUNOS	41
-#else _sys_types_h
+#else /* _sys_types_h */
 #ifdef NFDBITS
 #define	SUNOS	40
-#else NFDBITS
+#else /* NFDBITS */
 #define	SUNOS	35
-#endif NFDBITS
-#endif _sys_types_h
+#endif /* NFDBITS */
+#endif /* _sys_types_h */
 #endif
 
 #if SUNOS >= 40
@@ -188,4 +188,4 @@
 typedef	short	MPR_T;		/* type used for memory pixrect data */
 typedef	u_short	UMPR_T;		/* unsigned equivalent of MPR_T */
 
-#endif	pr_impl_util_DEFINED
+#endif	/* pr_impl_util_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pr_io.h xview-3.2p1.4/lib/pixrect/pr_io.h
--- xview-3.2p1.4.orig/lib/pixrect/pr_io.h	Tue Jun 29 07:18:34 1993
+++ xview-3.2p1.4/lib/pixrect/pr_io.h	Mon Aug 19 22:48:01 2002
@@ -52,4 +52,4 @@
 extern Pixrect * pr_load_image();
 extern Pixrect * pr_load_std_image();
 
-#endif pr_io_DEFINED
+#endif /* pr_io_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pr_line.h xview-3.2p1.4/lib/pixrect/pr_line.h
--- xview-3.2p1.4.orig/lib/pixrect/pr_line.h	Tue Jun 29 07:18:34 1993
+++ xview-3.2p1.4/lib/pixrect/pr_line.h	Mon Aug 19 22:48:01 2002
@@ -47,7 +47,7 @@
 	int width;
     } Pr_brush;
 
-#endif pr_line_h_DEFINED 
+#endif /* pr_line_h_DEFINED */
 
 
 
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pr_planegroups.h xview-3.2p1.4/lib/pixrect/pr_planegroups.h
--- xview-3.2p1.4.orig/lib/pixrect/pr_planegroups.h	Tue Jun 29 07:18:35 1993
+++ xview-3.2p1.4/lib/pixrect/pr_planegroups.h	Mon Aug 19 22:48:01 2002
@@ -47,4 +47,4 @@
 extern void pr_set_plane_group();
 extern void pr_set_planes();
 
-#endif	pr_planegroups_DEFINED
+#endif	/* pr_planegroups_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/pr_util.h xview-3.2p1.4/lib/pixrect/pr_util.h
--- xview-3.2p1.4.orig/lib/pixrect/pr_util.h	Tue Jun 29 07:18:35 1993
+++ xview-3.2p1.4/lib/pixrect/pr_util.h	Mon Aug 19 22:48:01 2002
@@ -66,11 +66,11 @@
 		op;							\
 	};								\
 	asm("dbra	d6,label");
-#else mc68000
+#else /* mc68000 */
 #define	rop_slowloop(n, op) \
 	{ register int _loop = (n); \
 		while (--_loop >= 0) { op; } }
-#endif mc68000
+#endif /* mc68000 */
 
 #ifdef mc68010
 #define cases8(n, op)							\
@@ -88,9 +88,9 @@
 			cases8(0, op); \
 			case 0:	break; \
 		} }
-#else mc68010
+#else /* mc68010 */
 #define	rop_fastloop	rop_slowloop
-#endif mc68010
+#endif /* mc68010 */
 
 /*
  * Alloctype(datatype) allocates a datatype structure using calloc
@@ -133,6 +133,6 @@
 
 #ifndef KERNEL
 Pixrect *pr_makefromfd();
-#endif !KERNEL
+#endif /* !KERNEL */
 
-#endif pr_util_DEFINED
+#endif /* pr_util_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/traprop.h xview-3.2p1.4/lib/pixrect/traprop.h
--- xview-3.2p1.4.orig/lib/pixrect/traprop.h	Tue Jun 29 07:18:35 1993
+++ xview-3.2p1.4/lib/pixrect/traprop.h	Mon Aug 19 22:48:01 2002
@@ -23,4 +23,4 @@
 	int y0, y1;			/* top+bottom boundaries */
 };
 
-#endif traprop_DEFINED
+#endif /* traprop_DEFINED */
diff -Nur xview-3.2p1.4.orig/lib/pixrect/tv1var.h xview-3.2p1.4/lib/pixrect/tv1var.h
--- xview-3.2p1.4.orig/lib/pixrect/tv1var.h	Tue Jun 29 07:18:39 1993
+++ xview-3.2p1.4/lib/pixrect/tv1var.h	Mon Aug 19 22:48:01 2002
@@ -10,7 +10,7 @@
 #include <sys/types.h>
 #ifndef SVR4
 #include <sundev/tv1reg.h>
-#endif SVR4
+#endif /* SVR4 */
 #include <pixrect/pixrect.h>
 #include <pixrect/memvar.h>
 #include <pixrect/cg4var.h>
@@ -59,6 +59,6 @@
 #define ROUNDUP(val, gran) (((val) - 1 | (gran) - 1) + 1)
 #endif
 
-#endif	!KERNEL
+#endif	/* !KERNEL */
 
-#endif tv1var_DEFINED
+#endif /* tv1var_DEFINED */
diff -Nur xview-3.2p1.4.orig/util/xgettext/xgettext.h xview-3.2p1.4/util/xgettext/xgettext.h
--- xview-3.2p1.4.orig/util/xgettext/xgettext.h	Tue Jun 29 07:14:13 1993
+++ xview-3.2p1.4/util/xgettext/xgettext.h	Mon Aug 19 22:48:01 2002
@@ -12,7 +12,7 @@
 #include <sys/dir.h>
 #else
 #include <dirent.h>
-#endif SVR4
+#endif /* SVR4 */
 
 #include <sys/param.h>
 #include <sys/stat.h>
@@ -54,4 +54,4 @@
     struct list_head	*next_list;
 };
 
-#endif XGETTEXT_H_DEFINED
+#endif /* XGETTEXT_H_DEFINED */
